Sources/CoreFoundation/CFDateFormatter.c

```diff
@@ -122,21 +122,21 @@ static void _CFDateFormatterStripAMPMIndicators(UniChar **bpat, int32_t *bpatlen
     }
 }
 
-CFStringRef CFDateFormatterCreateDateFormatFromTemplate(CFAllocatorRef allocator, CFStringRef tmplate, CFOptionFlags options, CFLocaleRef locale) {
+CFStringRef CFDateFormatterCreateDateFormatFromTemplate(CFAllocatorRef allocator, CFStringRef template, CFOptionFlags options, CFLocaleRef locale) {
```

Comment:
Usually this kind of thing is to avoid the reserved C++ name.

Reply:
good point: reverted in 921591b24b2c9a21c4d3817950501d40e7b4c743